quora-service/src/main/java/com/upgrad/quora/service/business/CommonService.java

```diff
@@ -0,0 +1,51 @@
+package com.upgrad.quora.service.business;
+
+import com.upgrad.quora.service.dao.UserDao;
+import com.upgrad.quora.service.entity.UserAuthTokenEntity;
+import com.upgrad.quora.service.entity.UserEntity;
+import com.upgrad.quora.service.exception.AuthorizationFailedException;
+import com.upgrad.quora.service.exception.UserNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class CommonService {
+
+    @Autowired
+    private UserDao userDao;
+
+    /**
+     *
```

Comment:
Please add description about this method.

Reply:
Added